golang/lint.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: golangci-lint
+spec:
+  inputs:
+    params:
+    - name: package
+      description: base package (and its children) under validation
+    - name: flags
+      description: flags to use for the test command
+      default: --verbose
+    - name: version
+      default: golangci-lint version to use
+      default: "v1.16"
+    - name: GOOS
+      description: "operating system target (default: node operating system)"
+      default:
+    - name: GOARCH
+      description: "architecture target (default: node architecture)"
+      default:
+    resources:
+    - name: source
+      type: git
+      targetPath: src/${inputs.params.package}
+  steps:
+  - name: lint
+    image: golangci/golangci-lint:${inputs.params.version}
```

Comment:
Do you think there's any benefit in having the default include the `:`, which would allow users to request a version by digest (`version: @sha256:...`) or specify `""` to get `:latest` (less useful)

Reply:
ping, still see `version`'s default without the `:`

This'll also need an update to _default:_s in README.md